tests/sort_plugin.test.ts

```diff
@@ -375,18 +375,31 @@ describe("Basic Sorting", () => {
   });
 });
 
-describe("Trigger sort generic errors", () => {
+describe("Sorting allowDispatch", () => {
   const sheetId: UID = "sheet2";
+  beforeEach(() => {
+    model = new Model({ sheets: [{ id: sheetId, colNumber: 1, rowNumber: 6 }] });
```

Comment:
I wouldn't restrict the number of cols/rows. Even more to only one column. It's error prone
Here it might be ok because there're only 3 tests and it's not "far", but if there are more tests in the future...

And if one (future) test requires 2 columns, it would need to redefine the model